app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -38,10 +38,11 @@ import androidx.compose.ui.unit.dp
 import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
 
 // Constants
-private const val PLOT_PADDING = 16 // Breathing room around the plot
+private const val DOT_DIAMETER = 36
+// Breathing room around the plot, ensuring the dot never touches sides of screen.
+private const val PLOT_PADDING = DOT_DIAMETER / 2 + 4
```

Comment:
When changing PLOT_SIZE const from `300` to something much larger that doesn't fit on say a Pixel 7 Pro - I noticed the dot was touching the sides, so updating the PLOT_PADDING value to prevent this. 

Before | After
-- | -- 
<img width="245" alt="Screenshot 2024-02-04 at 6 30 10 PM" src="https://github.com/heidiproske/kodeco_android_bootcamp_coordplot/assets/6706831/83a4ec30-e1ad-4dbf-9c9b-7732022f5f37"> | <img width="250" alt="Screenshot 2024-02-04 at 6 31 44 PM" src="https://github.com/heidiproske/kodeco_android_bootcamp_coordplot/assets/6706831/be303210-2e02-4d24-8983-9f84ee51411d">